.swiftlint.yml

```diff
@@ -6,7 +6,8 @@ opt_in_rules:
 excluded:
 - Carthage
 - Pods
-- SwiftLint/Common/3rdPartyLib
+#- SwiftLint/Common/3rdPartyLib
+- DerivedData
```

Comment:
?

Reply:
Local builds sre crashing due to linting errors in DerivedData. 
3rdPartyLib is likely a bug